pom.xml

```diff
@@ -76,6 +80,11 @@
             <artifactId>mapstruct</artifactId>
             <version>${org.mapstruct.version}</version>
         </dependency>
+        <dependency>
```

Comment:
I don't think it will be possible to properly serialize and deserialize objects without it. Besides, an article you provided as an example also uses it.

Reply:
bet it's possible ;)